logstash-core/lib/logstash/api/commands/stats.rb

```diff
@@ -45,14 +61,14 @@ def events
           )
         end
 
-        def pipeline(pipeline_id = nil)
+        def pipeline(pipeline_id = nil, opts)
```

Comment:
Since `pipeline` is a public method, I would prefer we not change its signature in a breaking way since there may be other bits that rely on the ability to send `Stats#pipeline` with no arguments.

```suggestion
        def pipeline(pipeline_id = nil, opts={})
```